x/bet/keeper/place_bet.go

```diff
@@ -26,33 +26,18 @@ func (k Keeper) PlaceBet(ctx sdk.Context, bet *types.Bet, activeBetOdds []*types
 		return types.ErrOddsUIDNotExist
 	}
 
-	// check if provided active odds are valid
-	if !allActiveOddsExist(activeBetOdds, bet.SportEventUID, sportEvent.OddsUIDs) {
-		return types.ErrActiveOddsUIDsNotValid
-	}
-
 	// check minimum bet amount allowed
 	if bet.Amount.LT(sportEvent.BetConstraints.MinAmount) {
 		return types.ErrBetAmountIsLow
 	}
 
-	// calculate vig and validate min and max vig satisfaction
-	vig := types.CalculateVig(activeBetOdds)
-	if vig.IsNegative() ||
-		vig.GT(sportEvent.BetConstraints.MaxVig) ||
-		vig.LT(sportEvent.BetConstraints.MinVig) {
-		return sdkerrors.Wrapf(types.ErrVigIsOutOfRange, "accepted range is %s - %s",
-			sportEvent.BetConstraints.MinVig,
-			sportEvent.BetConstraints.MaxVig)
-	}
-
 	// modify the bet fee and subtracted amount
 	setBetFee(bet, sportEvent.BetConstraints.BetFee)
 
 	// calculate extraPayout
 	extraPayout := calculateExtraPayout(bet)
 
-	if err := k.sporteventKeeper.AddExtraPayoutToEvent(ctx, sportEvent, bet.OddsUID, bet.Amount, extraPayout); err != nil {
+	if err := k.sporteventKeeper.AddExtraPayoutToEvent(ctx, sportEvent.UID, extraPayout); err != nil {
```

Comment:
Do we need to add extra payout to event?

Reply:
removed